api/src/app/modules/analytics-dashboard/analytics-dashboard.controller.ts

```diff
@@ -0,0 +1,217 @@
+import {
+  Controller,
+  Get,
+  Query,
+  DefaultValuePipe,
+  ParseIntPipe,
+} from '@nestjs/common';
+import { ApiOperation, ApiResponse, ApiTags, ApiQuery } from '@nestjs/swagger';
+import { AnalyticsDashboardService } from '@api-modules/analytics-dashboard/analytics-dashboard.service';
+import { DateRangeRequest } from '@api-modules/analytics-dashboard/analytics-dashboard.dto';
+import {
+  ProjectCountByDistrictResponse,
+  ProjectCountByForestClientResponse,
+} from '@api-modules/project/project.dto';
+import {
+  PublicCommentCountByDistrictResponse,
+  PublicCommentCountByCategoryResponse,
+  PublicCommentCountByProjectResponse,
+} from '@api-modules/public-comment/public-comment.dto';
+
+@ApiTags('analytics-dashboard')
+@Controller('analytics-dashboard')
+export class AnalyticsDashboardController {
+  constructor(private readonly analyticsDashboardService: AnalyticsDashboardService) {}
+
+  /**
+   * Returns the total number of FOM projects submitted
+   * within the specified commenting open date range,
+   * excluding those with an INITIAL workflow status.
+   *
+   * @param query - DateRangeRequest containing startDate and endDate in 'YYYY-MM-DD' format
+   * @returns Number of FOM projects in the date range
+   */
+  @Get('project/count')
+  @ApiOperation({
+    summary: 'Get total number of FOMs submitted in a date range',
```

Comment:
This summary "FOMs submitted..." doesn't match the wording of the method or service method called. The other wordings imply that all FOMs in the date range are being returned, versus ones just in a particular state. 
As per other comments, we can't 100% accurately query for FOMS submitted (state = Finalized) in a date range because we don't capture the finalized date in the database.